### PR TITLE
[Merged by Bors] - chore: classify `was simp` porting notes

### PR DESCRIPTION
**Pull request by pitmonticone**
Classifies by adding issue number (#10745) to porting notes claiming `was simp`.

### CODE REVIEW

File: Mathlib/Data/IsROrC/Basic.lean [hunk 1]
```diff
@@ -181,7 +181,7 @@ theorem ofReal_inj {z w : ℝ} : (z : K) = (w : K) ↔ z = w :=
 #align is_R_or_C.of_real_inj IsROrC.ofReal_inj
 
 set_option linter.deprecated false in
-@[deprecated, isROrC_simps] -- porting note: was `simp` but `simp` can prove it
+@[deprecated, isROrC_simps] -- porting note (#10745): was `simp` but `simp` can prove it
```
Comment (Vierkantor) [inline Mathlib/Data/IsROrC/Basic.lean:184]: This looks more like a previous `@[simp]` annotation:
```suggestion
@[deprecated, isROrC_simps] -- porting note (#10618): was `simp` but `simp` can prove it
```
Same for the rest in this file.

File: Mathlib/SetTheory/Cardinal/Basic.lean [hunk 2]
```diff
@@ -1355,7 +1355,7 @@ theorem card_le_of_finset {α} (s : Finset α) : (s.card : Cardinal) ≤ #α :=
   @mk_coe_finset _ s ▸ mk_set_le _
 #align cardinal.card_le_of_finset Cardinal.card_le_of_finset
 
--- Porting note: was `simp`. LHS is not normal form.
+-- porting note (#10745): was `simp`. LHS is not normal form.
 -- @[simp, norm_cast]
```
Comment (Vierkantor) [inline Mathlib/SetTheory/Cardinal/Basic.lean:1359]: This is not quite #10618 but certainly it's a different class of note.

File: Mathlib/Topology/Algebra/Module/Basic.lean [hunk 3]
```diff
@@ -427,7 +427,7 @@ instance continuousSemilinearMapClass :
 /-- Coerce continuous linear maps to functions. -/
 --instance toFun' : CoeFun (M₁ →SL[σ₁₂] M₂) fun _ => M₁ → M₂ := ⟨DFunLike.coe⟩
 
--- porting note: was `simp`, now `simp only` proves it
+-- porting note (#10745): was `simp`, now `simp only` proves it
```
Comment (Vierkantor) [inline Mathlib/Topology/Algebra/Module/Basic.lean:430]:
```suggestion
-- porting note (#10618): was `simp`, now `simp only` proves it
```

**Review by Vierkantor — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/SetTheory/Cardinal/Basic.lean (hunk 2 above)
Comment (pitmonticone) [reply Mathlib/SetTheory/Cardinal/Basic.lean:1359]: Ok @Vierkantor, I'll remove the issue label.

Comment (pitmonticone) [reply Mathlib/SetTheory/Cardinal/Basic.lean]: Done.

**Pull request closed without merging**